### DISCUSSION TITLE
add custom payload to api_tokens table

### DISCUSSION THREAD

**Discussion by andriyandriyan** [adonisjs/core#3848] [Help] [answered]
How to add custom payload data like organization_id into api_tokens table ?

**Comment by Melchyore** [comment, marked as answer]
@andriyandriyan Hello, https://docs.adonisjs.com/guides/auth/api-tokens-guard#meta <- is this what you are looking for?

**Reply by andriyandriyan** [reply]
i have tried it, but i dont know how to get the data from auth

**Reply by Melchyore** [reply]
> i have tried it, but i dont know how to get the data from auth

What do you mean by "get the data from auth"? You want to get the organization_id from thr user object? If so, I think you should execute an update query on tokens table with user ID as condition.
